rpc/src/rpc_service.rs

```diff
@@ -525,7 +525,14 @@ impl JsonRpcService {
                 );
                 let server = ServerBuilder::with_meta_extractor(
                     io,
-                    move |_req: &hyper::Request<hyper::Body>| request_processor.clone(),
+                    move |req: &hyper::Request<hyper::Body>| {
+                        let xbigtable = req.headers().get("x-bigtable");
+                        if xbigtable.map(|v| v == "disabled").unwrap_or(false) {
```

Comment:
```suggestion
                        if xbigtable.is_some_and(|v| v == "disabled") {
```

Reply:
nice, did not know about `is_some_and`! `matches!` also would be a good alternative to `map+unwrap_or`